dicoogle/src/main/java/pt/ua/dicoogle/plugins/PluginController.java

```diff
@@ -483,7 +483,7 @@ public QueryInterface getQueryProviderByName(String name, boolean onlyEnabled){
         List<String> baseProviders = ServerSettingsManager.getSettings()
                 .getArchiveSettings().getDIMProviders();
 
-        if (baseProviders.isEmpty()) {
+        if (baseProviders==null || baseProviders.isEmpty()) {

```

Comment:
Good catch. I had the feeling that this wasn't working properly in my recent uses of Dicoogle 3.

Reply:
Yeap!